frontend/packages/console-dynamic-plugin-sdk/scripts/generate-doc.ts

```diff
@@ -41,3 +45,127 @@ renderTemplate('scripts/templates/console-extensions.md.ejs', {
   printComments: (docComments: string[]) => printJSDocComments(docComments).replace(/\n/g, '<br/>'),
   escapeTableCell: (value: string) => value.replace(/\|/g, '\\|'),
 });
+
+type ComponentInfo = {
+  name: string;
+  doc: any;
+};
+
+const EXAMPLE = '@example';
+const DYNAMIC_PKG_PATH = '@console/dynamic-plugin-sdk/';
+
+const renderDocNode = (docNode: tsdoc.DocNode): string => {
+  let result: string = '';
+  if (docNode) {
+    if (docNode.kind === 'LinkTag') {
```

Comment:
Hey @bipuladh, I was a little unclear why we aren't able to support links. Can you explain?

Reply:
We could support LinkTags. But this is something that requires a bit more of a discussion. As we are handling the rendering of `DocNode`s, we need to come up with a mechanism for how links are handled. For ex: Which part of the block forms the link and the anchor link text. 
However I just looked at API Extractor's doc and we could do something similar to them https://api-extractor.com/pages/tsdoc/tag_link/ and support it.